packages/myst-cli/src/utils/imagemagick.ts

```diff
@@ -187,14 +202,14 @@ export async function convertImageToWebp(
   const convertGif = makeExecutable(`gif2webp -q ${quality} "${image}" -o "${webp}"`, debugLogger);
   // Density has to be BEFORE the PDF
   const convertPdfPng = makeExecutable(
-    `magick -density 600 -colorspace RGB ${image} -trim ${png}`,
+    `${imageMagickCommand()} -density 600 -colorspace RGB ${image} -trim ${png}`,
     debugLogger,
   );
   const convertPdfWebP = makeExecutable(`cwebp -q ${quality} "${png}" -o "${webp}"`, debugLogger);
 
   try {
     if (path.extname(image) === '.pdf') {
-      if (!isImageMagickAvailable() && !isWebpAvailable()) {
+      if (!isImageMagickAvailable() || !isWebpAvailable()) {
```

Comment:
This was also a bug - both of these are required for pdf webp conversion; previously this passed if only one was available.